spec/board_spec.rb

```diff
@@ -1,10 +1,15 @@
+require './helpers/Helper'
 require '../ttt/Board'
```

Comment:
What I usually do is to load all the files in a `spec_helper.rb`. Like this:

```rb
Dir[File.join('.', 'src/**/*.rb')].sort.each do |f|
  require f
end
```

It's not always necessary but could be a time-saver if you have lots of files.

Reply:
will look into this...